### DISCUSSION TITLE
How to merge table cells

### DISCUSSION THREAD

**Discussion by hendra-foo** [ianstormtaylor/slate#5048] [Q&A]
In Plate playground there is a table with column merged, but i can't find any docs on how to do column merging.
Can anybody lend me a help?

**Comment by dylans** [comment]
There are colspan and rowspan attributes. In the playground example, click on the various source code files, e.g. tableValue.tsx

https://github.com/udecode/plate/blob/main/packages/nodes/table/src/createTablePlugin.ts#L56-L71 shows how these attributes get included.
